config.js

```diff
@@ -18,8 +18,16 @@ exports.create = (api) => {
 
     config = addSockets(config)
     config = fixLocalhost(config)
-    config = pubHopSettings(config)
-    config = torOnly(config)
+
+    try {
+      window
+      // HACK (mix): patch-settings currently uses localStorage as persistence
+      // this means trying to load config outside context of electron crashes app ):
+      config = pubHopSettings(config)
+      config = torOnly(config)
+    } catch (e) {
+      console.log('TODO: decouple patchbay config from localStorage')
```

Comment:
Seems like I'm hitting this one

Reply:
mmmm I think I might need to finally improve `patch-settings` to use actual config files instead of `localStorage`